### PR TITLE
Bump up 0.8.2

### PR DESCRIPTION
**Pull request by riseshia**
related to #53 

This pr includes:

- Updated assets

### CODE REVIEW

**Comment by riseshia** [comment]
@seapy Could you check this? it's my first bump up, so I'm glad to be reviewed from you! XD

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 0.8.2.0 (2016-11-03)
+
+Updates:
+
+  - Update assets to version 0.8.2
+
+Changes:
+
+  - [Support sprocket 3.x](summernote asset update to 0.8.://github.com/summernote/summernote-rails/issues/49)
```
Comment (seapy) [inline CHANGELOG.md]: link syntax is wrong

Comment (riseshia) [inline CHANGELOG.md]: 😱

**Comment by riseshia** [comment]
Updated XD

**Pull request merged**